frontend/src/lib/utils.tsx

```diff
@@ -238,6 +238,11 @@ export const dateTimeOperatorMap: Record<string, string> = {
     is_not_set: '✕ is not set',
 }
 
+export const relativeDateTimeOperatorMap: Record<string, string> = {
+    is_relative_date_before: '< relative date before',
+    is_relative_date_after: '> relative date after',
```

Comment:
Why not let the user select "time before" or even just "time range", and then either select a specific date or a range? The existing insight date picker could even work:

<img width="342" alt="image" src="https://github.com/PostHog/posthog/assets/53387/3d0bbcd4-9de2-4dfd-b5ae-4f46c030a554">

Reply:
heh, I did not think about this, but makes a lot of sense? Will update, we _should_ be able to detect based on whether it's a timestamp or a relative date time.